deep_qa/common/util.py

```diff
@@ -30,3 +30,42 @@ def add_noise_to_dict_values(dictionary: Dict[Any, float], noise_param: float) -
         noise = random.uniform(-noise_value, noise_value)
         new_dict[key] = value + noise
     return new_dict
+
+
+def clean_layer_name(input_name: str,
+                     keep_right_of_last_backslash: bool=None,
```

Comment:
Right, I was just saying that taking the right hand side after splitting is currently unused, so we might as well not add the complexity if it's not needed.